src/playwright-runner.js

```diff
@@ -263,6 +262,13 @@ async function run (nodeBin, runCfgPath, suiteName) {
     throw new Error(`Could not find projectPath directory: '${projectPath}'`);
   }
 
+  process.env.PLAYWRIGHT_CFG_FILE = runCfg.playwright.configFile || '';
+  process.env.BROWSER_NAME = suite.param.browserName;
+
+  if (suite.param.project) {
+    process.env.project = suite.param.project;
```

Comment:
Are you sure this env var is needed? Isn't project being defined as a command line arg?

Reply:
removed 😄